xml/release-notes.xml

```diff
@@ -334,6 +334,27 @@
     <package>libzypp</package> versions older than 12.25.8:
    </para>
    <screen>zypper dup --allow-vendor-change --force-resolution</screen>
+
+   <para>
+    &opensuseleap; 15.3 supplies all required rpm veryfication keys including the &sles;
+    as part of <package>openSUSE-build-key</package>.
+    All keys are also newly available in the oss repository.
+
+    <package>libzypp-17.25.11</package> imports all keys automatically.
+    However, we recommend following steps in case you need to import keys manually.
```

Comment:
Before this part, we should provide a way how to check if it's needed
rpm -qa gpg-pubkey # should list the SLE-15 key gpg-pubkey-39db7c82-5847eb1f